swap_meet/clothing.py

```diff
@@ -1,2 +1,7 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
```

Comment:
Since Item and Clothing are in the same swap_meet package, we can import the class with shorter syntax `from .item import Item`. If they were not in the same package, then we would need to specify the package name `swap_meet`